src/components/containers/Sidebar/Sidebar.css

```diff
@@ -1,9 +1,31 @@
 .sidebar {
-  /* background-color: cadetblue; */
-  width: 30%;
+  width: 40%;
   align-items: center;
-  background-color: lightgrey;
-  border-radius: 1rem;
-  height: 35rem;
-  margin: 2rem;
+  height: 10rem;
+  margin: 1rem;
+  padding: 5px;
+}
+
+.sidebar:hover {
+}
+
+button:hover {
+  background-color: darkmagenta;
+}
+
+.IconSizes {
+  height: 9rem;
+  width: 100%;
```

Comment:
We should probably make sure the height and width are either both constant (some number of rem) or both percentages of the parent. Otherwise, these icons are going to stretch when the browser size changes

Reply:
Changed